server/pages/index.jade

```diff
@@ -115,6 +115,7 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				}
 			})();
 
+		script(src='/calypso/polyfill.js')
```

Comment:
should append the hash here, just in case we have to change it. maybe tie into the other static bits in `index.js`?